x-pack/plugin/shutdown/src/main/java/org/elasticsearch/xpack/shutdown/TransportGetShutdownStatusAction.java

```diff
@@ -254,11 +253,20 @@ static ShutdownShardMigrationStatus shardMigrationStatus(
         int totalRemainingShards = relocatingShards + startedShards + initializingShards;
 
         // If there's relocating shards, or no shards on this node, we'll just use the number of shards left to move
-        if (relocatingShards > 0 || totalRemainingShards == 0) {
-            SingleNodeShutdownMetadata.Status shardStatus = totalRemainingShards == 0
-                ? SingleNodeShutdownMetadata.Status.COMPLETE
-                : SingleNodeShutdownMetadata.Status.IN_PROGRESS;
-            return new ShutdownShardMigrationStatus(shardStatus, startedShards, relocatingShards, initializingShards);
+        if (totalRemainingShards == 0) {
```

Comment:
Is there a change here or just a different preference for how to write this?

Reply:
This is a trivial refactoring to remove duplicated condition